### PR TITLE
[refactor](stream-load) Adjust the content of the review  Added direct load via BeNodes

### PR DESCRIPTION
**Pull request by MYiYang**
[fateure](stream-load) Added direct load via BeNodes
[refactor](stream-load) Adjust the content of the review

# changes
Added the mode of displaying BENODES configuration for stream-load operation.
Configure doris.benodes. The doris.fenodes configuration needs to be deleted.
Such as 0.0.0.0:8040, X.X.X.X: 8040.

### CODE REVIEW

File: spark-doris-connector/src/main/java/org/apache/doris/spark/rest/RestService.java [hunk 1]
```diff
@@ -592,17 +650,20 @@ public static List<BackendV2.BackendRowV2> getBackendRows(SparkSettings sparkSet
...
-        List<BackendV2.BackendRowV2> backends = getBackendRows(sparkSettings, logger);
-        Collections.shuffle(backends);
-        BackendV2.BackendRowV2 backend = backends.get(0);
-        return backend.getIp() + ":" + backend.getHttpPort();
+    private static List<BackendV2.BackendRowV2> getBackendRowV2s(SparkSettings sparkSettings, Logger logger) throws DorisException {
+        List<BackendV2.BackendRowV2> backends = null;
+        if(notBeNode(sparkSettings, logger)){
+            backends = getBackendRows(sparkSettings, logger);
+        }else{
+            backends = getBeBackendRows(sparkSettings, logger);
```
Comment (JNSimba) [inline spark-doris-connector/src/main/java/org/apache/doris/spark/rest/RestService.java:664]: Is it enough to call the `getBeBackendRows` method directly in the `getBackendRows` method? Other methods should not need to be modified?

File: spark-doris-connector/src/main/java/org/apache/doris/spark/rest/RestService.java [hunk 2]
```diff
@@ -560,6 +582,42 @@ static List<BackendRow> parseBackend(String response, Logger logger) throws Dori
         return backendRows;
     }
 
+    /**
+     * get Doris BE node.
+     * @param logger slf4j logger
+     * @return the Doris BE node
+     * @throws IllegalArgumentException BE nodes is illegal
+     */
+    public static List<BackendV2.BackendRowV2> getBeBackendRows(SparkSettings sparkSettings,  Logger logger) throws DorisException {
```
Comment (JNSimba) [inline spark-doris-connector/src/main/java/org/apache/doris/spark/rest/RestService.java:591]: Backend means be. Maybe we can change the function name?

File: spark-doris-connector/src/main/java/org/apache/doris/spark/rest/RestService.java (hunk 1 above)
Comment (MYiYang) [reply spark-doris-connector/src/main/java/org/apache/doris/spark/rest/RestService.java:664]: Yeah, I should be able to take a look

File: spark-doris-connector/src/main/java/org/apache/doris/spark/rest/RestService.java (hunk 2 above)
Comment (MYiYang) [reply spark-doris-connector/src/main/java/org/apache/doris/spark/rest/RestService.java:591]: Ok, let me think about what I should call it

**Pull request closed without merging**